onnxruntime/core/session/inference_session.h

```diff
@@ -593,6 +618,15 @@ class InferenceSession {
   // Flag indicating if ModelProto has been parsed in an applicable ctor
   bool is_model_proto_parsed_ = false;
   const Environment& environment_;
+
+  // Bytes from an ORT format model.
+  // We store them currently to make the Load + Initialize behave the same way as for an ONNX model
+  // as we need some of the bytes for the Load (create the Model) and some for the Initialize (create SessionState).
+  // Short term we free them after Initialize.
+  // Longer term we may want to directly refer to offsets in this buffer for initializers so we don't need to copy
+  // those into new OrtValue instances, at which point we won't free them until the InferenceSession goes away.
+  size_t ort_format_model_num_bytes_ = 0;
+  std::unique_ptr<uint8_t[]> ort_format_model_bytes_;
```

Comment:
Intention was to support a Load API where the model bytes are provided in-memory and we take ownership of those without a copy. But possibly need memory allocated by an OrtAllocator as uint8_t[] won't work for the C API. I'll change to a vector for now but may change again depending on if/when we support that sort of load (in mobile scenarios memory usage is more important so that could be sooner rather than later).